serialization/compact/src/main/java/CompactFullConfig.java

```diff
@@ -0,0 +1,51 @@
+import com.hazelcast.config.CompactSerializationConfig;
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.nio.serialization.compact.CompactReader;
+import com.hazelcast.nio.serialization.compact.CompactSerializer;
+import com.hazelcast.nio.serialization.compact.CompactWriter;
+
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * Demonstrates how to use Compact(BETA) serialization from java in a polyglot setup where hazelcast clients written in
+ * different languages needs to be used.
+ */
+public class CompactFullConfig {
```

Comment:
```suggestion
public class CompactProgrammaticConfig {
```

Reply:
I want this to be not example of ProgrammaticConfig. This is Zero Config vs ? . And I think Full Config is a good name for ?
What do you think ?